uint64 timestamp		# time since system start (microseconds)
int32 return_value
uint32 return_sequence
